app/templates/events/view/tickets/access-codes/create.hbs

```diff
@@ -0,0 +1,81 @@
+<div class="ui stackable grid">
+  <div class="row">
+    <div class="ui header column">
+      {{t 'Create an access code'}}
+    </div>
+  </div>
+  <div class="row">
+    <div class="sixteen wide column">
+      <form class="ui form">
+        <div class="field">
+          <label class="required">{{t 'Access Code'}}</label>
+          {{input type='text' name='access_code'}}
+        </div>
+        <div class="field">
+          <label class="required">{{t 'Number of Access Tickets'}}</label>
+          {{input type='number' name='number_of_access_tickets' value=0}}
+        </div>
+
+        <div class="grouped inline fields">
+          <label class="required">{{t 'Status'}}</label>
+          <div class="field">
+            {{ui-radio name="status" label="Active" value="active"}}
+          </div>
+          <div class="field">
+            {{ui-radio name="status" label="Inactive" value="inactive"}}
+          </div>
+        </div>
+        <div class="field">
+          <label class="required">{{t 'Select Tickets applied to the access code'}}</label>
+          <div class="ui checkbox">
+            <input type="checkbox" name="ticket_types">
```

Comment:
Please use ember semantic UI checkbox through out the project instead of the default one, thanks.
http://semantic-org.github.io/Semantic-UI-Ember/#/modules/checkbox